s3torchconnector/src/s3torchconnector/_s3client/_mock_s3client.py

```diff
@@ -25,11 +25,13 @@ def __init__(
         bucket: str,
         user_agent: Optional[UserAgent] = None,
         s3client_config: Optional[S3ClientConfig] = None,
+        path_style: Optional[bool] = False,
```

Comment:
Could we move this parameter to be an attribute on `S3ClientConfig` instead if passing it through raw?

Reply:
Let me check the feasibility and update the PR.